package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "knockout.autocomplete",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
Assuming the two commits following the version increase in 966cf15 are also part of the feature, you should at least rebase the commit and append it to the end.

However, it's not optimal to version/tag inside a feature branch. If you don't rebase and fast-forward your branch on master, your tag will not represent the master branch in a particular point in time.

It might not be relevant this time, but it is when you are working on a feature while other features land on master.

--

I think that's what you want to do:

Let's assume this PR is complete and CI passed, and you are on a feature branch called `feature/append-autocomplete` in the local checkout of the `One-com/knockout.autocomplete` project repository.

_To create a branch of this name, do_
```
cd One-com/knockout.autocomplete
git checkout -b feature/append-autocomplete master
git pull https://github.com/mikkeldamm/knockout.autocomplete.git master
```

Then to do the merge
```
git checkout master
git merge --no-ff feature/append-autocomplete
```

```
npm version patch // or major or minor
```
This command effectively does: bumping the version patch number in your package.json, then `commit -am "v[New version number]"` and then creates an [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging#Annotated-Tags) `git tag -a v[New version number] -m "[New version number]"`

Finally, you can do
```
git push --follow-tags
```
This does `git push` and `git push --tags`

--
From your question I think this might be relevant, excuse me if I'm explaining what you already know!